### PR TITLE
remove the clientset adapter

### PR DESCRIPTION
**Pull request by deads2k**
This removes the clientset adapter entirely.  There is one focused adapter that remains in a single e2e test.  Every other reference was removed.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33281)

### CODE REVIEW

**Comment by deads2k** [comment]
@caesarxuchao Alternate approach.  Let's just remove it entirely.

File: test/e2e/framework/framework.go [hunk 1]
```diff
@@ -58,7 +59,12 @@ const (
 type Framework struct {
 	BaseName string
 
-	Client        *client.Client
+	// Client is manually created and should not be used unless absolutely necessary.  Use Clientset_1_5
```
Comment (caesarxuchao) [inline test/e2e/framework/framework.go]: extra space before "Use".

File: test/e2e/framework/framework.go [hunk 2]
```diff
@@ -58,7 +59,12 @@ const (
...
+	// where possible.
+	Client *client.Client
+	// ClientSet uses internal objects, you should use Clientset_1_5 where possible.
```
Comment (caesarxuchao) [inline test/e2e/framework/framework.go]: nit: change "ClientSet" to "Clientset"? Or change "Clientset_1_5" to "ClientSet_1_5". If the change requires more than grep+sed then I can live with the mismatch.

File: test/integration/framework/master_utils.go [hunk 3]
```diff
@@ -324,64 +322,13 @@ func ScaleRC(name, ns string, replicas int32, restClient *client.Client) (*api.R
...
 	}
-	scaled, err := restClient.ReplicationControllers(ns).Get(name)
+	scaled, err := clientset.Core().ReplicationControllers(ns).Get(name)
 	if err != nil {
 		return nil, err
 	}
 	return scaled, nil
 }
 
-// StartRC creates given rc if it doesn't already exist, then updates it via kubectl's scaler.
```
Comment (caesarxuchao) [inline test/integration/framework/master_utils.go]: These are just dead code?

**Review by caesarxuchao — COMMENTED** [review]
Just two nits. Thanks @deads2k!

File: test/integration/framework/master_utils.go (hunk 3 above)
Comment (deads2k) [inline test/integration/framework/master_utils.go]: > These are just dead code?

Yeah.

File: test/e2e/framework/framework.go (hunk 2 above)
Comment (deads2k) [inline test/e2e/framework/framework.go]: > nit: change "ClientSet" to "Clientset"? Or change "Clientset_1_5" to "ClientSet_1_5". If the change requires more than grep+sed then I can live with the mismatch.

Ok.  I'll see what I can on Monday.

**Comment by deads2k** [comment]
renamed and removed space.

**Comment by deads2k** [comment]
@k8s-bot gce node test this

**Comment by deads2k** [comment]
@k8s-bot test this: issue #33380

**Comment by caesarxuchao** [comment]
Thanks, @deads2k!

**Pull request merged**